src/controllers/trigger.js

```diff
@@ -27,6 +28,7 @@ const AUDITS = {
   lhs, // for all lhs variants
   404: notfound,
   'broken-backlinks': backlinks,
+  'organic-keywords': keywords,
```

Comment:
For all of the keys here, we could probably reuse the types exported from the audit model